src/Eurofurence.App.Server.Services/DependencyResolution/AutofacModule.cs

```diff
@@ -69,6 +71,15 @@ private void RegisterConfigurations(ContainerBuilder builder)
 
         private void RegisterServices(ContainerBuilder builder)
         {
+            var connectionString = _configuration.GetConnectionString("Eurofurence");
+            var dbContextOptionsBuilder = new DbContextOptionsBuilder<AppDbContext>();
+            dbContextOptionsBuilder.UseMySql(connectionString, ServerVersion.AutoDetect(connectionString));
+
+            builder
```

Comment:
```csharp
builder.AddDBContext<AppDbContext>(options =>
{
    var connectionString = _configuration.GetConnectionString("Eurofurence");
    options.UseMySql(connectionString, ServerVersion.AutoDetect(connectionString));
});
```

Reply:
Thanks for the suggestion, sadly in this case we are using autofac instead of the Microsoft pipeline so there is no AddDbContext. And the autofac module is used in two places (startup and the CLI toolbox) so I didn't want to refactor that yet. But I opened a separate issue to check if we can throw out autofac in favor of the Microsoft pipeline, so I would address this there

Reply:
Ah okay! Yeah, would be good since `AddDBContext` adds more than just the DB context, but that is for another time then! Looks good!